actors/common/src/main/java/org/sunbird/common/quartz/scheduler/ManageCourseBatchCount.java

```diff
@@ -25,6 +27,11 @@
  */
 public class ManageCourseBatchCount implements Job {
 
+  private Map<String, Map<String, Object>> contentMap = new HashMap<>();
```

Comment:
do not use states in job -> ideally job should be stateless.

Use variable passing - if you require - but initalize this at start of execute and as soon as execute method is over - this should get garbage collected